### PR TITLE
Disable auto detect schema activity bits

### PR DESCRIPTION
**Pull request by jdpgrailsdev**
## What
* Disable auto detect schema bits that prevent data plane from running

## How
* Disable usage of `ConfigFetchActivity`

### CODE REVIEW

File: airbyte-workers/src/test/java/io/airbyte/workers/temporal/sync/SyncWorkflowTest.java [hunk 1]
```diff
@@ -255,6 +256,7 @@ void testReplicationFailure() {
   }
 
   @Test
+  @Disabled
```
Comment (alovew) [inline airbyte-workers/src/test/java/io/airbyte/workers/temporal/sync/SyncWorkflowTest.java:259]: do we need to disable all these? I think we can just remove verifyShouldRefreshSchema and verifyRefreshSchema

**Review by alovew — APPROVED** [review]

**Review by benmoriceau — APPROVED** [review]

File: airbyte-workers/src/test/java/io/airbyte/workers/temporal/sync/SyncWorkflowTest.java (hunk 1 above)
Comment (jdpgrailsdev) [reply airbyte-workers/src/test/java/io/airbyte/workers/temporal/sync/SyncWorkflowTest.java]: @alovew You are correct.  We figured that out after doing the blanket disable.  I believe that we only ended up disabling one test and just commented out the references to those verifications in the others.

**Review by git-phu — APPROVED** [review]

**Comment by octavia-approvington** [comment]
After a careful ML study, <br/> we think this looks okay.  <br/> imagine code being okay

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**